### ISSUE TITLE
2021-1-3

### ISSUE THREAD

**Issue by ssfc** [ssfc/wangdao#413]
551Reinstall windows; (2020-12-25)
552Install wamp; done; install acrobat; done; install VScode; done; install Pycharm; done; (2020-12-25)
553Set github desktop proxy; [http]     proxy = socks5://127.0.0.1:9909    (2020-12-25)
554Upload necessary software to baiduyun; upload wampserver; upload github desktop; upload anaconda3; upload Acrobat; upload VScode; upload Pycharm; upload free download manager; (2020-12-25)
555Review LSTM; find Andrw Ng’s material and sample of LSTM; review Andrew Ng’s machine learning course; register a new account using HUST edu account; (2020-12-25)
556Install tensorflow 1.15 on anaconda3; done; download and install git in pycharm; done; (2020-12-26)
557run sample program linear regression; try to install matplotlib; done; (2020-12-26)
558Why gradient descent make target function smaller, not larger? 从数学上的角度来看，梯度的方向是函数增长速度最快的方向，那么梯度的反方向就是函数减少最快的方向; (2020-12-26)
559黑塞矩阵（德语：Hesse-Matrix；英语：Hessian matrix 或 Hessian），又译作海森矩阵、海塞（赛）矩阵或海瑟矩阵等，是一个由多变量实值函数的所有二阶偏导数组成的方块矩阵，由德国数学家奥托·黑塞引入并以其命名; (2020-12-26)
